src/ActionBar/ActionBar.js

```diff
@@ -1,31 +1,39 @@
-import 'fundamental-styles/dist/action-bar.css';
 import ActionBarActions from './_ActionBarActions';
 import ActionBarBack from './_ActionBarBack';
 import ActionBarHeader from './_ActionBarHeader';
 import classnames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
+import withStyles from '../utils/WithStyles/WithStyles';
 
-const ActionBar = ({ children, className, ...props }) => {
+const ActionBar = React.forwardRef(({ children, className, customStyles, disableStyles, ...props }, ref) => {
```

Comment:
I think we can get rid of the `customStyles` in this prop pruning use case since this never gets passed here from `withStyles`.  It should stay in the PropTypes definition, however, because consumers are unaware that the component is wrapped in an HOC.